test/Interop/SwiftToCxx/functions/swift-primitive-functions-cxx-bridging.swift

```diff
@@ -4,8 +4,8 @@
 
 // RUN: %check-interop-cxx-header-in-clang(%t/functions.h)
 
-// CHECK:      inline float passThrougCFloat(float x) noexcept SWIFT_WARN_UNUSED_RESULT {
-// CHECK-NEXT: return _impl::$s9Functions16passThrougCFloatyS2fF(x);
+// CHECK:      inline float passThroughCFloat(float x) noexcept SWIFT_WARN_UNUSED_RESULT {
+// CHECK-NEXT: return _impl::$s9Functions16passThroughCFloatyS2fF(x);
```

Comment:
It's not enough to respell things in mangled names. When corrections are made, the lengths of these strings can change, and substitutions may be introduced which will completely alter the textual part of the name.

Reply:
Sorry, I do understand the technical bits. For now, I'm dropping these items. I won't have time to address them otherwise for a while.